src/WCSimConstructRealisticPlacement.cc

```diff
@@ -317,6 +317,7 @@ G4LogicalVolume* WCSimDetectorConstruction::ConstructRealisticPlacement()
   // and light tight volumes. PMTs are then placed in the corresponding
   // volumes using G4Assembly classes.
 
+    bool checkOverlapsPMTRealistic = false;
```

Comment:
Please add this as a data member [here](https://github.com/WCSim/WCSim/blob/develop/include/WCSimDetectorConstruction.hh#L503), and set based on the cmake option. We're using this overlap checking in the CI, and so it is useful to have a way to turn it on/off

If it really is useful to have the realistic PMT overlap checking set differently to the other overlap checking, then please add as a new cmake option (just a minor bit of copy paste)

Or actually, it may make most sense to add a new CMake option e.g. `WCSIM_CHECK_PMT_GEOMETRY_OVERLAPS`, and use `checkOverlapsPMT` everywhere (no new `checkOverlapsPMTRealistic` variable)